data/os/AIX.yaml

```diff
@@ -0,0 +1,8 @@
+---
+rsyslog::package_name: rsyslog.base
+rsyslog::group_name: system
+rsyslog::service_name: syslogd
+rsyslog::switch_default_syslog: true
+# you will need to fill the source out 
+# aix has no yum like service
+rsyslog::package_source: '<package_location>'
```

Comment:
This should have a real value like `/opt/freeware/src/packages` or `/tmp` so that the module can be tested.

Reply:
I don't think this is possible, because it is up to the user to put the file on the system.  It could be on nfs or any other directory and is up to the user to specify where that is.  I could just put /tmp but this will likely be wrong for the user but I suppose the way it is now is also wrong.